decidim-initiatives/app/views/decidim/initiatives/initiatives/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <%
 edit_link(
```

Comment:
If we want to remove the edit link from these pages, shouldn't we remove the whole `edit_link` call from these places?

Reply:
Makes sense, thanks!